lib/createWorkers.js

```diff
@@ -48,10 +50,37 @@ function createWorkers(handlers, config, options = {}) {
 
   return {
     listen: co.wrap(_listen),
-    close: co.wrap(_close)
+    close: co.wrap(_close),
+    get emitter() {
+      return emitter;
+    },
+    wait(key) {
```

Comment:
In a second time, I would return a specific value based on the triggered event. For instance, we might return the task return value on `task.completed` or the reason for `task.retried`.